src/components/sections/landing/HeroSection.astro

```diff
@@ -75,15 +78,17 @@ interface Props {
   </div>
   <!-- Hero Image Section -->
   <div class="flex w-full">
-    <div class="top-12 overflow-hidden">
-      <Image
-        src={heroImage}
-        alt="Stack of ScrewFast product boxes containing assorted hardware tools"
+      <div class="top-12 overflow-hidden">
+	{imageSrc && imageAlt &&
+	<Image
+        src={imageSrc}
+        alt={imageAlt}
         class="h-full w-full scale-110 object-cover object-center"
         draggable={"false"}
         loading={"eager"}
-        format={"avif"}
-      />
+	 />
```

Comment:
Also adding back `format={"avif"}` for our case. By default, the <Image /> component will produce a .webp file.

Reply:
In my opinion, we can just keep it in the Image component of a Hero section as it was.

Reply:
ok